docs/content/preview/api/ysql/the-sql-language/statements/cmd_copy.md

```diff
@@ -143,3 +143,11 @@ The following copy options may help to speed up copying, or allow for faster rec
 * `DISABLE_FK_CHECK` skips the foreign key check when copying new rows to the table.
 * `REPLACE` replaces the existing row in the table if the new row's primary/unique key conflicts with that of the existing row.
```

Comment:
For REPLACE , can we document the limitation reported in the use case -->  https://github.com/yugabyte/yugabyte-db/issues/13687#issuecomment-1219960469

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/13677/commits/2cdcec07c6831e68d64f3daa04bfee9b8d9e4fda